### PR TITLE
Fix az linux CLI command

### PR DESCRIPTION
**Pull request by stuartatmicrosoft**

### CODE REVIEW

**Comment by DanyC97** [comment]
@stuartatmicrosoft maybe you want to add docs in the PR's title and also couple of words in the description - e.g _fixing a typo in the azure config docs or s'thing_

0.002$ though

File: docs/user/azure/credentials.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can request permissions using the Azure portal or the Azure cli.
 Find the AppId for your service principal by using,
 
 ```console
-$ az ad sp list --show-mine -otable
+$ az ad sp list --show-mine -o table
```
Comment (abhinavdahiya) [inline docs/user/azure/credentials.md]: the previous incantation also works correctly.. so why this change?

**Comment by abhinavdahiya** [comment]
> https://github.com/openshift/installer/pull/2323#discussion_r321812494

/close

Please feel to reopen if you think it the comment is not accurate.
